packages/components/src/angle-picker-control/index.tsx

```diff
@@ -48,10 +49,9 @@ function UnforwardedAnglePickerControl(
 
 	const classes = clsx( 'components-angle-picker-control', className );
 
-	const unitText = <UnitText>°</UnitText>;
-	const [ prefixedUnitText, suffixedUnitText ] = isRTL()
-		? [ unitText, null ]
-		: [ null, unitText ];
+	const prefixOrSuffixProp = isRTL()
+		? { prefix: <InputControlPrefixWrapper>°</InputControlPrefixWrapper> }
+		: { suffix: <InputControlSuffixWrapper>°</InputControlSuffixWrapper> };
```

Comment:
By the way, while we are here, is it really correct to use either prefix or suffix depending on RTL? I'd expect that "prefix" and "suffix" are semantic terms, something like CSS logical properties (`*-start`, `*-end`), and that the `NumberControl` would take care of displaying them in the right order.

Reply:
Thanks for reminding me, there should be a code comment here (169ecc3f1998f4f932f9725b675b5793d5a0cc68). I was actually confused too and looked up the history to make sure.

The prefix and suffix slots reposition themselves "semantically" by default, but we're intentionally overriding that behavior here.